static/src/javascripts/bootstraps/enhanced/newsletters.js

```diff
@@ -44,6 +44,15 @@ const addSubscriptionMessage = (buttonEl: HTMLButtonElement): void => {
     });
 };
 
+const modifyDataLinkName = (modifier: string) => (el: HTMLButtonElement) : void => {
+    const firstStageName = el.getAttribute('data-link-name') || "undefined-data-link-name"
+    el.setAttribute('data-link-name', firstStageName + modifier)
+}
+
+const modifyAtSecondStage = (el: HTMLButtonElement) => modifyDataLinkName('-second-stage')(el)
```

Comment:
Can we make it explicit in the name what we're modifying? e.g. `modifyLinkNameForSecondStage`